cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -754,38 +754,148 @@
 
   <property>
     <name>kafka.default.replication.factor</name>
+    <value>${kafka.server.default.replication.factor}</value>
+    <description>
+      CDAP Kafka service replication factor (deprecated: replaced with kafka.server.replication.factor)
+    </description>
+  </property>
+
+  <property>
+    <name>kafka.log.dirs</name>
+    <value>${kafka.server.log.dirs}</value>
+    <description>
+      CDAP Kafka service log storage directory (deprecated: replaced with kafka.server.log.dir)
+    </description>
+  </property>
+
+  <property>
+    <name>kafka.log.flush.interval.messages</name>
+    <value>${kafka.server.log.flush.interval.messages}</value>
+    <description>
+      The interval length at which will force an fsync of data written to the log (deprecated:
+      replaced with kafka.server.log.flush.interval.messages)
+    </description>
+  </property>
+
+  <property>
+    <name>kafka.log.retention.hours</name>
```

Comment:
We don't need this because the `kafka.log.retention.hours` was not a public config that user can change.